plugins/filter/ipsubnet.py

```diff
@@ -252,73 +252,88 @@ def _ipsubnet(*args, **kwargs):
     return ipsubnet(**updated_data)
 
 
-def ipsubnet(value, query="", index="x"):
+def ipsubnet(value, query="", index=None):
     """ Manipulate IPv4/IPv6 subnets """
 
-    try:
-        vtype = ipaddr(value, "type")
-        if vtype == "address":
-            v = ipaddr(value, "cidr")
-        elif vtype == "network":
-            v = ipaddr(value, "subnet")
-
-        value = netaddr.IPNetwork(v)
-    except Exception:
+    vtype = ipaddr(value, "type")
+    if not vtype:
         return False
-    query_string = str(query)
+    elif vtype == "address":
+        v = ipaddr(value, "cidr")
+    elif vtype == "network":
+        v = ipaddr(value, "subnet")
+    value = netaddr.IPNetwork(v).cidr
+    vtype = ipaddr(value, "type")
+
     if not query:
         return str(value)
```

Comment:
I realize this isn't even something you've touched, but while we're in here we should always be returning `to_text()` instead of `str()`